vuepress/docs/next/docs/getting-started/ent-bundle.md

```diff
@@ -2,150 +2,181 @@
+|`ent bundle publish --registry [registry]`| Publish the Docker images to a specified Docker registry |
 
-## ECR Commands
-Entando provides a series of `ent ecr` commands for managing bundle interactions with the [Entando Component Repository](../../docs/compose/local-hub-overview.md) (ECR). 
+### Deploy
+| Command| Description
+|:--|:--
+|`ent bundle deploy`| Deploy a bundle to the Local Hub of an Entando Application |
+
+### Install
+| Command| Description
+|:--|:--
+|`ent bundle install`| Install a bundle in the Local Hub of an Entando Application |
+
+## ECR Commands 
+The following commands are applicable to both docker-based and git-based bundles.
 
 | Command| Descriptions
 |:--|:--
 |`ent ecr deploy`| Generate the custom resource (CR) and deploy it to the current profile |
-|`ent ecr gen-secret`| Generate and display a plugin Secret skeleton|
-|`ent ecr generate-cr`|Generate the custom resource |
-|`ent ecr get-bundle-id [repository-url]` | Calculate and display the bundle ID
-|`ent ecr get-plugin-code --auto --repo=[repository-url]` |Calculate and display the plugin code for a Git-based bundle|
-|`ent ecr get-plugin-code [component-name] --repo=[repository-url]` | Calculate and display the plugin code for a Docker-based bundle
-|`ent ecr install`| Install a bundle to the ECR|
-|`ent ecr install --conflict-strategy=OVERRIDE`|Adopt a strategy for conflicts on installed bundles|
-|`ent ecr list`| Display the list of bundles associated with the current profile|
-|`ent ecr uninstall`| Uninstall a bundle|
+|`ent ecr gen-secret`| Generate and display the skeleton for a plugin Secret |
+|`ent ecr generate-cr`| Generate the custom resource |
+|`ent ecr get-bundle-id [repository-url]` | Calculate and display the bundle ID |
+|`ent ecr get-plugin-code` | Calculate and display the plugin code |
+|`ent ecr install`| Install a bundle in the ECR |
+|`ent ecr install --conflict-strategy=OVERRIDE`| Adopt a strategy for conflicts affecting installed bundles |
+|`ent ecr list`| Display the list of bundles associated with the current profile |
+|`ent ecr uninstall`| Uninstall a bundle |
 
 #### Command Details
-* `ent ecr get-bundle-id` and `ent ecr get-plugin-id`: Entando uses a unique identifier for a bundle as a way to provide customization parameters and add security controls for bundle-specific resources. A unique identifier is also generated for each microservice plugin in your project.
+* `ent ecr get-bundle-id`: The unique identifier assigned to each bundle provides a mechanism to customize parameters and add security controls for bundle-specific resources
 
-* `ent ecr install --conflict-strategy=OVERRIDE`: If a project bundle has already been installed, the `--conflict-strategy` flag forces a `CREATE`, `SKIP`, or `OVERRIDE` strategy for components.
+* `ent ecr get-plugin-code`: 
+   * Uniquely and safely identifies each bundle plugin and derived cloud-native resources
```

Comment:
this wording is borrowed from notion: https://www.notion.so/entando/Plugin-Code-former-pluginId-a273ff8c3b80479284fdc9e6744b66f1

Reply:
I think this is confusing wording since it implies the implementation. Maybe  "Get the unique identifier for the bundle plugin". The only important bit there is that it's unique, really. Some place we may need to note that the plugin-code changes if you move the repo/registry but this is probably the wrong place for it.